src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+
+    public Figure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        color = colorSupplier.getRandomColor();
+    }
```

Comment:
let's create a constructor with the parameter `String color` instead of this one and call it in subclasses of `Figure`